packages/app-store/routing-forms/pages/router/getServerSideProps.ts

```diff
@@ -152,12 +160,20 @@ export const getServerSideProps = async function getServerSideProps(
       response,
       serializableForm.fields
     );
+
     return {
       redirect: {
         destination: getAbsoluteEventTypeRedirectUrl({
           eventTypeRedirectUrl: eventTypeUrlWithResolvedVariables,
           form: serializableForm,
-          allURLSearchParams: new URLSearchParams(stringify(context.query)),
+          allURLSearchParams: getUrlSearchParamsToForward({
```

Comment:
Reusing `getUrlSearchParamsToForward` instead that we use when use headed router. It ensures same query params area added in headless and headed modes.